data/stories/get_started_intro.yml

```diff
@@ -0,0 +1,172 @@
+version: "2.0"
+stories:
+
+- story: Getting started with Rasa playground
+  steps:
+  - intent: get_started_intro
+  - action: utter_greet
+  - action: utter_ask_intro
+  - action: utter_ask_playground_or_install
+  - intent: affirm
+  - action: utter_what_help
```

Comment:
I think we need to follow a similar flow also for `how_to_get_started`; e.g. things like `try it out` in response to  `utter_ask_playground_or_install` currently will start the old get started flow. We could split up that intent into `get_started_intro` (e.g. try it out, how do i build a bot, etc) and `install_rasa` (and maybe keep the rest where it is?) so that there's less confusion between which getting started flow the user follows